string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,10 @@
+/* exported ransomCase */
+/* make the word all lowercase;
+look at each letter in the word and capiltalize every other letter starting at teh second letter;
+return the output of the new word */
+function ransomCase(string) {
+  var newString = string.toLowerCase();
+  for (var i = 1; i < string.length; i += 2) {
+    newString = newString.substring(0, i) + newString[i].toUpperCase() + newString.substring(i + 1);
+  } return newString;
```

Comment:
Make sure your `return` statement is always on its own line, it should not be on the same line as a closing brace.